### ISSUE TITLE
where is the video saved?

### ISSUE THREAD

**Issue by DittmerOk** [WWBN/AVideo#7672]
please tell me where the video is saved? I tried to trace, but as a result, the video is saved to the AVideo / videos folder and also to the AVideo / AVideo-Encoder / videos folder. But with different names. What is the difference? I understand that the originals are added to the AVIdeo/ENcoder/videos folder. And in the avideo/videos folder ... already convected files. Is it possible to make it so that the original videos are not loaded into the encoder folder?

**Comment by DanielnetoDotCom** [comment]
the video names are random unique names. there is no way to predict it.

**Comment by DittmerOk** [comment]
yes ... I understand that. But why then upload the original video to the AVideo / Encoder / videos folder. If it is not used anywhere? And the video that is played on the site is at Avideo/videos?
